sysbuild/Kconfig.suit

```diff
@@ -40,6 +40,67 @@ config SUIT_ENVELOPE_SIGN_SCRIPT
 	  - --output-file <STRING> - location of signed envelope to create by script
 	default "modules/lib/suit-generator/ncs/sign_script.py"
 
+choice SUIT_ENVELOPE_ENCRYPT_SCRIPT
+	prompt "Select the used SUIT encryption script"
+	help
+	   The SUIT encryption script is used to generate encryption artifacts for images inside a SUIT envelope.
+	   It is the "external" script that is called by the build system - it is responsible for creating
+	   various needed SUIT structures, but passes the actual encryption of the payload to a separate KMS script.
+	default SUIT_ENVELOPE_ENCRYPT_SCRIPT_DEFAULT
+
+config SUIT_ENVELOPE_ENCRYPT_SCRIPT_DEFAULT
+	bool "Use the default SUIT encryption script"
+	help
+	  Use the default encryption script to generate encryption artifacts for images inside a SUIT envelope.
+
+endchoice
+
+config SUIT_ENVELOPE_ENCRYPT_SCRIPT_PATH
+	string "Location of SUIT encrypt script"
+	help
+	  Python script called to generate encryption artifacts for images inside a SUIT envelope.
+	  See the help message for the default encryption script to see what arguments the script
+	  must accept.
+	default "${ZEPHYR_NRF_MODULE_DIR}/../modules/lib/suit-generator/ncs/encrypt_script.py" if SUIT_ENVELOPE_ENCRYPT_SCRIPT_DEFAULT
```

Comment:
use the actual module name instead of a path from the nrf module?

Reply:
Ok, changed